docker/ccf-pdo.debugging.yaml

```diff
@@ -0,0 +1,47 @@
+# Copyright 2018 Intel Corporation
```

Comment:
Did you test this image?

Reply:
It's not an image. It's the equivalent of what we had with Sawtooth (see here https://github.com/hyperledger-labs/private-data-objects/blob/a95abcf2892c61d96b02cc2026f7137efada174b/docker/sawtooth-pdo.debugging.yaml) to enable the `PDO_DEBUG_BUILD` feature in docker compose. I can't recall if used it in my tests.